### PR TITLE
feat: add check to make sure ALBs log to a CBS satellite bucket

### PR DESCRIPTION
**Pull request by maxneuvians**
This PR adds a rule to check if ALBs are logging access requests to a CBS satellite bucket. It also adds the required CI integration using a custom script until the `-d` action is fixed `cfn-guard`.

### CODE REVIEW

File: .devcontainer/post-create-command.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Install Guard CLI
+curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/aws-cloudformation/cloudformation-guard/main/install-guard.sh | sh
```
Comment (CalvinRodo) [inline .devcontainer/post-create-command.sh:4]: While I have no issue with this why isn't this baked into the container?

**Review by CalvinRodo — APPROVED** [review]
:taco::cat:

File: .devcontainer/post-create-command.sh (hunk 1 above)
Comment (patheard) [inline .devcontainer/post-create-command.sh:4]: much secure, so safe

Comment (maxneuvians) [reply .devcontainer/post-create-command.sh:4]: Not a feature in the devcontainer.json, we are dockerfileless around here

**Pull request merged**